### PR TITLE
Fix React 'unique key prop' errors

### PR DESCRIPTION
**Pull request by delaaxe**
Fix classic 'Each child in a list should have a unique "key" prop' errors

### CODE REVIEW

File: src/components/BalancesList.js [hunk 1]
```diff
@@ -169,15 +169,14 @@ export default function BalancesList() {
...
-        );
-      });
+      const key = pk.toString();
+      const Memoized = React.memo(() => (
+        <BalanceListItem
+          publicKey={pk}
+          setUsdValue={setUsdValuesCallback}
+        />
+      ));
+      return {key, Memoized};
```
Comment (gotjoshua) [inline src/components/BalancesList.js:179]: can you explain the syntax here (just for my curiosity)

i see that u are mapping all the pks from an array... but than means you are returning an array of objects from useMemo, right?

```js
[
 {key0, Memoized0},
 {key1, Memoized1},
...
]
```
is this a strong perf optimization?

Comment (axeldelamarre) [reply src/components/BalancesList.js:179]: {a, b} is syntactic sugar for {a: a, b: b}
No optimization at all, I just needed to pass the key down below in the JSX tree.
But now that I think of it I can probably rewrite this more cleanly

Comment (axeldelamarre) [reply src/components/BalancesList.js]: Ok done, should be simpler now

File: src/components/BalancesList.js [hunk 2]
```diff
@@ -169,14 +169,13 @@ export default function BalancesList() {
...
     return sortedPublicKeys.map((pk) => {
-      const key = pk.toString();
       const Memoized = React.memo(() => (
         <BalanceListItem
           publicKey={pk}
           setUsdValue={setUsdValuesCallback}
         />
       ));
-      return {key, Memoized};
+      return <Memoized key={pk.toString()}/>;
```
Comment (gotjoshua) [inline src/components/BalancesList.js:178]: I like it. Nice simplification.

**Review by gotjoshua — APPROVED** [review]
just wanted to comment actually not start a review...

**Comment by axeldelamarre** [comment]
@jhlx @armaniferrante ok for merge?

**Pull request closed without merging**